src/VivView.js

```diff
@@ -0,0 +1,77 @@
+import { OrthographicView } from '@deck.gl/core';
+
+/**
+ * This class generates a layer and a view for use in the VivViewer
+ * @param {number} viewState The viewState object
+ * @param {string} id The id for the current view
+ * @param {string} x The x location on the screen for the current view
+ * @param {string} y The y location on the screen for the current view
+ */
+export default class VivView {
+  constructor({ viewState, x = 0, y = 0 }) {
+    const { height, width, id } = viewState;
+    this.width = width;
+    this.height = height;
+    this.initViewState = viewState;
+    this.id = id;
+    this.x = x;
+    this.y = y;
+  }
+
+  /**
+   * Create a boudning box from a viewport based on passed-in viewState.
+   * @param {viewState} Object The viewState for a certain viewport
+   * @returns {View} The DeckGL View for this viewport
+   */
+  static makeBoundingBox(viewState) {
+    const viewport = new OrthographicView().makeViewport({
+      // From the current `detail` viewState, we need its projection matrix (actually the inverse).
+      viewState,
+      height: viewState.height,
+      width: viewState.width
+    });
+    // Use the inverse of the projection matrix to map screen to the view space.
+    return [
+      viewport.unproject([0, 0]),
+      viewport.unproject([viewport.width, 0]),
+      viewport.unproject([viewport.width, viewport.height]),
+      viewport.unproject([0, viewport.height])
+    ];
+  }
```

Comment:
Again somewhat incline to have this be a `util`. I generally prefer static methods for offering convenience methods to end user (i.e. creating class instances `VivView.fromConfig(configObj) --> new VivView` or if you think people would be inclined to use:

```javascript
import { VivView } from 'viv';

const boundingBox = VivView.makeBoundingBox(state);
```

Or if you foresee others creating inherited classes. In that case keep, but make sure to use `this.makeBoundingBox` on the children otherwise its the same as importing a function..

Reply:
Ok.